examples/step-92/doc/results.dox

```diff
@@ -0,0 +1,497 @@
+
+  # cells output data filename
+  set cells output data filename = example-1_cells_output_data
+
+end
+@endcode
+
+Next section shows how to generate the mesh without opening the gmsh GUI,
+how to start the simulation and a graphical visualization of the output data.
+
+<h4> results </h4>
+
+Generate a coarse mesh in binary file format, run the program and see the output
+@code
+> /usr/bin/gmsh example-1.geo -clmax 0.3 -bin -3
+> mpirun -n 32 -bind-to hwthread ./step-92 example-1.prm
+Running with PETSc on 32 MPI rank(s)...
+mesh-file: "example-1.msh"
+setup: number of degrees of freedom: 15015
+set default boundary potential: phi_e_0 = 0 V
+set terminal_11 boundary potential: phi_e_11 = 1 V
+set terminal_12 boundary potential: phi_e_12 = 0.5 V
+set terminal_21 boundary potential: phi_e_21 = 0.5 V
+set terminal_22 boundary potential: phi_e_22 = -1 V
+  system set up.
+  system assembled.
+  solved.
+exported cells solution and output data into file "example-1_cells_output_data.vtu"
+
+
++---------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start    |     0.636s |            |
+|                                             |            |            |
+| Section                         | no. calls |  wall time | % of total |
++---------------------------------+-----------+------------+------------+
+| 01. make grid                   |         1 |     0.229s |        36% |
+| 02. setup                       |         1 |    0.0142s |       2.2% |
+| 03. assemble                    |         1 |   0.00516s |      0.81% |
+| 04. solve                       |         1 |     0.339s |        53% |
+| 05. post-process                |         1 |    0.0395s |       6.2% |
++---------------------------------+-----------+------------+------------+
+@endcode
+note: The mpirun-parameters <code>-n 32 -bind-to hwthread</code>
+are essential on the PC used for preparing this tutorial,
+featuring an AMD 7950X CPU with 16 cores and 32 hardware-threads.
+The number of hardware-threads has to be adopted to the number of
+CPUs hardware-threads on the PC of the reader.
+
+
+In this tutorial, ParaView was used to visualize the solver output.
+Light blue lines show electric field lines ($\vEfield$). Colorization of conductor and barrier surfaces
+indicate the electric potential $\Epot$ on those surfaces.
+Yellow lines show electric current density $\vJelfield$.
+
+<table width="100%" align="center">
+  <tr>
+      <td align="center">
+      <img src="../../../examples/step-92/doc/example-1_field_plot_1.jpg" alt="example-1: 3D visualization of electric field lines and current density" height="800"/>
+      <p> example-1: 3D visualization of electric flux lines (light blue), current density (yellow lines) and electric potential on solids surfaces (using ParaView)</p>
+      </td>
+  </tr>
+</table>
+
+
+<h3> example-2 </h3>
+
+<h4> geometry and parameters </h4>
```

Comment:
You have used this header before in line 5 of this file. It seems like doxygen doesn't like that: it is creating section labels for the html version based on the title, and assumes they are unique. Can you try a different name?